src/Data/Abstract/ModuleTable.hs

```diff
@@ -21,9 +21,7 @@ import Prologue
 import System.FilePath.Posix
 
 newtype ModuleTable a = ModuleTable { unModuleTable :: Map.Map ModulePath a }
-  deriving stock (Eq, Foldable, Functor, Generic1, Generic, Ord, Traversable)
-  deriving newtype (Lower, Monoid, Semigroup)
-  deriving anyclass (NFData)
```

Comment:
Not deriving `NFData` means not needing `-XDeriveAnyClass` which means not needing `-XDerivingStrategies`.